src/test/scala/temple/generate/target/openapi/OpenAPIGeneratorTest.scala

```diff
@@ -178,6 +178,20 @@ class OpenAPIGeneratorTest extends FlatSpec with Matchers {
         |          $ref: '#/components/responses/Error401'
         |        '500':
         |          $ref: '#/components/responses/Error500'
+        |    get:
+        |      summary: Look up the single match associated with the access token
+        |      tags:
+        |      - Match
+        |      responses:
+        |        '302':
+        |          description: The single match is accessible from the provided Location
+        |          headers:
+        |            Location:
+        |              description: The location where the single match can be found
+        |              schema:
+        |                type: string
+        |        '404':
+        |          $ref: '#/components/responses/Error404'
```

Comment:
Shouldn't this endpoint only generate for services with an auth block? i.e. `User` not `Match`

Apologies if this test differs from the spec-golang model

Reply:
Yeah the "match" name is somewhat incidental, the test just generates every possible endpoint for a service with every field type.

Reply:
The logic to decide when to generate this endpoint isn't yet implemented - this is evident from the lack of changes to any `E2E` test 😄